2025-03-28 10:40:20,319 [INFO] AutoDBImporter.py:261 - Ê≠£Âú®ÂàõÂª∫Ë°® data_mzmx...
2025-03-28 10:40:20,376 [ERROR] AutoDBImporter.py:287 - ÊâπÂ§ÑÁêÜÈîôËØØ: Execution failed on sql '
        SELECT
            name
        FROM
            sqlite_master
        WHERE
            type IN ('table', 'view')
            AND name=?;
        ': not all arguments converted during string formatting
2025-03-28 10:40:20,378 [ERROR] AutoDBImporter.py:296 - Êìç‰ΩúÂ§±Ë¥•: Execution failed on sql '
        SELECT
            name
        FROM
            sqlite_master
        WHERE
            type IN ('table', 'view')
            AND name=?;
        ': not all arguments converted during string formatting
2025-03-28 10:40:20,380 [ERROR] AutoDBImporter.py:310 - Á≥ªÁªüÈîôËØØ: Execution failed on sql '
        SELECT
            name
        FROM
            sqlite_master
        WHERE
            type IN ('table', 'view')
            AND name=?;
        ': not all arguments converted during string formatting
2025-03-28 10:40:20,402 [INFO] _internal.py:225 - 127.0.0.1 - - [28/Mar/2025 10:40:20] "[35m[1mPOST /api/upload HTTP/1.1[0m" 500 -
2025-03-28 10:42:21,641 [INFO] _internal.py:225 -  * Restarting with stat
2025-03-28 10:42:22,973 [WARNING] _internal.py:225 -  * Debugger is active!
2025-03-28 10:42:22,976 [INFO] _internal.py:225 -  * Debugger PIN: 144-678-273
2025-03-28 10:42:23,207 [INFO] _internal.py:225 -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2025-03-28 10:42:40,610 [INFO] _internal.py:225 - 127.0.0.1 - - [28/Mar/2025 10:42:40] "GET /api/sql-configs HTTP/1.1" 200 -
